.github/workflows/test.yaml

```diff
@@ -55,6 +55,10 @@ jobs:
         run: |
           pip install jupyter_packaging wheel jupyterlab
           python setup.py sdist bdist_wheel
+      - name: Check Package Sizes
+        run: |
+          ls -lathr dist
+          find dist -type f -size +200k | grep dist || echo ok
```

Comment:
Is this just for info, or should it fail if there's a file > 200k ? At the moment the exit code will always be 0 regardless.

Reply:
yeah, i need to figure out how to make it fail properly. will try some more stuff locally.

Reply:
should be fully resolved by 33b424c